frontend/src/components/housing/enquiry/EnquiryIntakeForm.vue

```diff
@@ -19,6 +20,13 @@ import { confirmationTemplate } from '@/utils/templates';
 
 import type { Ref } from 'vue';
 
+const { formUpdated, stopAutoSave } = useAutoSave(async () => {
+  const values = formRef.value?.values;
+  if (values) {
+    await onSaveDraft(values, true);
+  }
+});
```

Comment:
I'm gonna keep the async and await as I have defined the composable callback function with `saveCallback: () => Promise<void>...` and am choosing to keep this so that future uses of the auto save composable allow for async callbacks.